docs/user/InstructionsForAutoTuner.md

```diff
@@ -30,7 +30,7 @@ which is a free minimal installer for the package manager `conda`.
 
 ```shell
 # set up conda environment
-conda create -n autotuner_env python=3.9
+conda create -n autotuner_env python=3.8 -y
 conda activate autotuner_env
 
 # install requirements
```

Comment:
A simple `setup.sh` would be helpful here. Users can use a single command to set up, and CI can use the same.

Reply:
`. autotuner_env/bin/activate`